bin/yaml/cpp.yaml

```diff
@@ -228,15 +228,25 @@ compilers:
         mips:
           - arch_prefix: "{subdir}-unknown-linux-gnu"
             check_exe: "{arch_prefix}/bin/{arch_prefix}-g++ --version"
-            targets:
-              - name: 5.4.0
-                subdir: mips
-              - name: 5.4.0
-                subdir: mipsel
-              - name: 5.4.0
-                subdir: mips64
-              - name: 5.4.0
-                subdir: mips64el
+            mips:
+              subdir: mips
+              targets:
+                - name: 5.4.0
+                - name: 11.2.0
+            mipsel:
+              subdir: mipsel
+              targets:
+                - name: 5.4.0
+            mips64:
+              subdir: mips64
+              targets:
+                - name: 5.4.0
+                - name: 11.2.0
+            mips64el:
+              subdir: mips64el
+              targets:
+                - name: 5.4.0
+
```

Comment:
is this legal yaml, an extra linebreak?

Reply:
Hmmm, don't know, but the script is not reporting any error. But will remove it if you find it disturbing !

Reply:
Done :)